llvm/test/Transforms/InferAddressSpaces/SPIRV/self-phi.ll

```diff
@@ -0,0 +1,29 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
You don't need to duplicate all of these tests. You just need some basic samples that the target is implemented, the full set is testing pass mechanics which can be done on any target

Reply:
Done.